### PR TITLE
Fixing APPCLOUD-176 (Close the ResultSets in Dao Classes in tier API)

### PR DESCRIPTION
**Pull request by kasundsilva**

### CODE REVIEW

File: modules/webapps/appCloudTierapi/src/main/java/org/wso2/appcloud/tierapi/util/DBUtil.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class DBUtil {
```
Comment (hisamith) [inline modules/webapps/appCloudTierapi/src/main/java/org/wso2/appcloud/tierapi/util/DBUtil.java]: Why did you create a new class? Can't we simply use org.wso2.appcloud.core.DBUtil class?

File: modules/webapps/appCloudTierapi/src/main/java/org/wso2/appcloud/tierapi/dao/impl/ContainerSpecDaoImpl.java [hunk 2]
```diff
@@ -69,29 +66,26 @@
 	public List<ContainerSpecifications> getContainerSpecByRuntimeID(int runtimeId) throws SQLException {
 		Connection dbConnection = null;
 		PreparedStatement preparedStatement = null;
+		ResultSet rs = null;
 		List<ContainerSpecifications> containerSpecsList = new ArrayList<ContainerSpecifications>();
 		try {
-			DBConfiguration dbCon = new DBConfiguration();
+			DBUtil dbCon = new DBUtil();
```
Comment (hisamith) [inline modules/webapps/appCloudTierapi/src/main/java/org/wso2/appcloud/tierapi/dao/impl/ContainerSpecDaoImpl.java]: :-1:
This is wrong! Creating a new Util class everytime???  :shit: :shit: :shit: :shit:

Comment (danushkaf) [inline modules/webapps/appCloudTierapi/src/main/java/org/wso2/appcloud/tierapi/dao/impl/ContainerSpecDaoImpl.java]: It was everywhere in tierapi. Fixed with https://github.com/wso2/app-cloud/pull/468

Comment (kasundsilva) [inline modules/webapps/appCloudTierapi/src/main/java/org/wso2/appcloud/tierapi/dao/impl/ContainerSpecDaoImpl.java]: Lets use app dbutil in core here as well.

**Pull request merged**